python/cuml/test/test_input_utils.py

```diff
@@ -344,3 +345,21 @@ def get_input(type, nrows, ncols, dtype, order='C', out_dtype=False):
                                 order=order)
     else:
         return result, np.array(cp.asnumpy(rand_mat), order=order)
+
+
+def test_tocupy_missing_values_handling():
```

Comment:
Can you add tests with fail_on_nan=True ?

Reply:
Thank you for the review. Done